Makefile

```
@@ -1,4 +1,3 @@
-DC := $(shell which docker-compose)
 DOCKER := $(shell which docker)
```

We're already relying on the environment path, so the `which` call is kinda pointless here.
```suggestion
DOCKER := "docker"
```